src/vs/workbench/contrib/terminal/browser/terminalEditor.ts

```diff
@@ -188,7 +190,11 @@ export class TerminalEditor extends EditorPane {
 	}
 
 	layout(dimension: dom.Dimension): void {
-		this._editorInput?.terminalInstance?.layout(dimension);
+		const instance = this._editorInput?.terminalInstance;
+		if (instance) {
+			instance.attachToElement(this._overflowGuardElement!);
```

Comment:
could `this._overflowGuardElement!` be undefined and throw?

Reply:
Not in practice as `createEditor` must have been called before `layout`. Not the best code but I'd rather not do clean up as part of this change.